src/app/datasets/page.tsx

```diff
@@ -171,7 +174,7 @@ export default function DatasetPage() {
             <div className="col-start-0 col-span-12 xl:col-span-8 xl:col-start-5">
               <DatasetList datasets={response.datasets!} />
             </div>
-            <div className="col-start-0 col-span-12 mt-20 xl:col-span-8 xl:col-start-5">
+            <div className="col-start-0 col-span-12 mb-8 mt-10 xl:col-span-8 xl:col-start-5 ">
```

Comment:
important: can you check if the footer is okay in different pages? I was expecting the fix to be on the footer component instead.

Reply:
fixed